python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import tensorflow as tf
+from tensorflow.keras.layers import Embedding as TFEmbedding
+
+
+class Embedding(TFEmbedding):
+
+    def __init__(self,
+                 input_dim,
+                 output_dim,
+                 embeddings_initializer='uniform',
+                 regularizer=None,
+                 embeddings_constraint=None,
+                 mask_zero=False,
+                 input_length=None,
+                 **kwargs):
```

Comment:
I think we should make the interface compatible with `keras.layers.Embedding`; that is, it accepts both `embeddings_regularizer` and `activity_regularizer`. If `embeddings_regularizer` is set, it will report warning, and use it as `activity_regularizer` (if the latter is not set).

Reply:
Ok, I'll change it in a follow up pr.